dev_scripts/pytest-wrapper.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# PyTest Wrapper script - temporary solution to tests failing non-deterministically [1]
+# This is only fixed in podman v4.3.0. The wrapper essentially runs the tests in sequence
+# when the podman version is lower.
+#
+# [1]: https://github.com/freedomofpress/dangerzone/issues/217
+
+# FIXME this whole script should be removed and the respective Makefile calling line
+# replaced once all supported platforms have a podman version >= v4.3.0.
+
+import re
+import subprocess
+import sys
+
+import pytest
+from pkg_resources import parse_version
+
+from dangerzone.container import get_runtime_name
+
+PODMAN_MIN_VERSION = "4.3.0"
+
+
+def get_podman_version():
+    result = subprocess.run(
+        "podman version --format '{{.Client.Version}}'", shell=True, capture_output=True
+    )
```

Comment:
Not critical, but it's best not to use `shell=True` unless necessary. In this setting, it's not a problem, but it builds muscle memory.

(For a reason why, see https://docs.python.org/3/library/subprocess.html#security-considerations)

Reply:
Yes, I'm aware. In this case in particular, the string was a constant and it's more readable. But we can also do it as you suggest. I have changed it in [5a61fff](https://github.com/freedomofpress/dangerzone/pull/288/commits/5a61fff68ae10b0c8fa84e2cf84b04853dc74f7b).